scripts/split-e2e-tests.ts

```diff
@@ -4,7 +4,7 @@ import { join } from 'path';
 import * as fs from 'fs-extra';
 import { supportedRegions } from '../packages/amplify-category-geo/src/constants';
 
-const CONCURRENCY = 12;
+const CONCURRENCY = 25;
```

Comment:
this isn't used anymore now that everything runs in parallel right?

Reply:
I don't want to remove the logic that uses this in case we get so many tests that we want to add waves again. So the number is purposefully set high enough that it runs it all in parallel